modules/helm/Makefile.repo

```diff
@@ -59,25 +59,26 @@ helm\:repo\:fix-perms:
 helm\:repo\:package:
 	$(call assert-set,REPO_PATH)
 	@echo "## Building packages in $(shell basename $(CURDIR))"
-	@find ./$(REPO_PATH)/library -maxdepth 1 -mindepth 1 -type d '!' -name packages  | \
+	@find ./$(REPO_PATH) -maxdepth 1 -mindepth 1 -type d '!' -name packages  | \
     	xargs -n 1 -I '{}' sh -c 'echo && echo "# Updating dependencies for {}" && $(HELM) dependency build --debug {}'
-	@find ./$(REPO_PATH)/library -maxdepth 1 -mindepth 1 -type d '!' -name packages  | \
+	@find ./$(REPO_PATH) -maxdepth 1 -mindepth 1 -type d '!' -name packages  | \
     	xargs -n 1 -I '{}' sh -c 'echo && echo "# Building package for {}" && $(HELM) package --debug {}'
+	@mkdir -p ./packages/$(REPO_PATH)
 	@find . -maxdepth 1 -mindepth 1 -type f -name '*.tgz' | \
-		xargs -I '{}' mv {} ./$(REPO_PATH)/packages
+		xargs -I '{}' mv {} ./packages/$(REPO_PATH)
 
 .PHONY : helm\:repo\:index
 helm\:repo\:index:
 	$(call assert-set,REPO_PATH)
 	@echo "## Generating index for $(CURRENT_REPO_URL)"
-	@$(HELM) repo index ./$(REPO_PATH)/packages --url $(CURRENT_REPO_URL)/$(REPO_PATH) --debug
+	@$(HELM) repo index ./packages/$(REPO_PATH) --url $(CURRENT_REPO_URL)/$(REPO_PATH) --debug
 
 .PHONY : helm\:repo\:lint
-## Lint library charts
+## Lint charts
 helm\:repo\:lint:
 	$(call assert-set,REPO_PATH)
-	@echo "## Linting charts $(shell pwd)/library"
-	@find ./$(REPO_PATH)/library -maxdepth 1 -mindepth 1 -type d | \
+	@echo "## Linting charts $(shell pwd)"
+	@find ./$(REPO_PATH) -maxdepth 1 -mindepth 1 -type d | \
```

Comment:
I don't think this should be prefixed with `./`; that is what `REPO_PATH` is for